Source/Core/Core/NetworkCaptureLogger.cpp

```diff
@@ -49,4 +69,155 @@ NetworkCaptureType BinarySSLCaptureLogger::GetCaptureType() const
 {
   return NetworkCaptureType::Raw;
 }
+
+PCAPSSLCaptureLogger::PCAPSSLCaptureLogger()
+{
+  std::time_t t = std::time(nullptr);
```

Comment:
You can use `fmt` for this to write this in a more concise way, see https://fmt.dev/latest/api.html#date-and-time-formatting